src/components/common/TmBalance.vue

```diff
@@ -170,8 +217,7 @@ export default {
       stakingDenom: "",
       sentToGA: false,
       balances: [],
-      selectedTokenFiatValue: `Tokens Total Fiat Value`,
-      selectedFiatCurrency: `EUR`, // EUR is our default fiat currency
+      selectedFiatCurrency: `EUR`, // EUR is our default fiat currency,
```

Comment:
is it necessary to have a default fiat? i imagined we would default to `crypto` and the user could select a fiat they like which would persist for them.

Reply:
Hm, interesting. Maybe a default fiat currency works for the API but not so for the FE.

So what should we show then in the fiat value bubbles by default? If we show nothing by default, how can the users know by a simple glimpse they can activate that display by selecting a currency in the selector?

Maybe add a `span` next to the currency selector when it is the first time and the user hasn't selected yet any?

Really good idea storing the preferred fiat currency! 👏